src/components/aside/AddRestaurantModal.jsx

```diff
@@ -0,0 +1,58 @@
+import styles from '../../css/Modal.module.css';
+import { CATEGORYOPTION } from '../constants/CategoryOption';
+const AddRestaurantModal = () => {
```

Comment:
```suggestion
import { CATEGORYOPTION } from '../constants/CategoryOption';

const AddRestaurantModal = () => {
```
요렇게 공백을 추가하는건 어떨까요??

Reply:
추가 하는게 좋겠네요 ! !